nfs/src/rpc_task.cpp

```diff
@@ -68,6 +68,89 @@ do { \
 #define INJECT_BAD_COOKIE(res, task) /* nothing */
 #endif
 
+#ifdef ENABLE_PRESSURE_POINTS
+#define INJECT_CREATE_FH_POPULATE_FAILURE(res, task) \
+do { \
+    /* \
+     * Must be called only for create and mknod task. \
+     */ \
+    assert(task->get_op_type() == FUSE_CREATE); \
+    if (res && (NFS_STATUS(res) == NFS3_OK)) { \
```

Comment:
let's inject only when originally it's not missing, i.e. check for (res)->CREATE3res_u.resok.obj.handle_follows